zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -732,6 +732,8 @@ public String getZeppelinSearchTempPath() {
     ZEPPELIN_NOTEBOOK_STORAGE("zeppelin.notebook.storage",
         "org.apache.zeppelin.notebook.repo.GitNotebookRepo"),
     ZEPPELIN_NOTEBOOK_ONE_WAY_SYNC("zeppelin.notebook.one.way.sync", false),
+    ZEPPELIN_NOTEBOOK_NEW_FORMAT_CONVERT("zeppelin.notebook.new_format.convert", false),
+    ZEPPELIN_NOTEBOOK_NEW_FORMAT_DELETE_OLD("zeppelin.notebook.new_format.delete_old", false),
```

Comment:
If zeppelin.notebook.new_format.convert=true and zeppelin.notebook.new_format.delete_old=false, would Zeppelin try to convert same old notes many times, after each restart?

Reply:
That's right, user need to set `zeppelin.notebook.new_format.convert` to false when converting is done. so that next time zeppelin won't do the conversion again.

Reply:
If the user forgot to change it back, every restart would convert an old version of an old format (json) and overwrites it on a new version of a newer version note. I would go with the script, but it's not easy when you have 100 notes in HDFS :(